onebusaway-android/src/androidTest/java/org/onebusaway/android/util/test/UIUtilTest.java

```diff
@@ -552,6 +553,75 @@ public void testArrivalInfoLabels() {
         assertEquals("On time", arrivalInfo.get(29).getStatusText());
         assertEquals("9 min delay", arrivalInfo.get(30).getStatusText());
         assertEquals("On time", arrivalInfo.get(31).getStatusText());
+
+        /**
+         * Test notification texts
+         */
+
+        assertEquals("Route " + arrivalInfo.get(0).getInfo().getShortName()
```

Comment:
Need to be updated to use `UIUtils.getRouteDisplayName()` instead of `getShortName()`.